src/setup/node.rs

```diff
@@ -202,17 +244,16 @@ impl Drop for Node {
     }
 }
 
-pub fn build_stateful_path() -> io::Result<PathBuf> {
+fn get_stateful_node_path() -> io::Result<PathBuf> {
+    // TODO support multiple nodes
     let ziggurat_path = build_ripple_work_path()?;
-    Ok(ziggurat_path.join(NODE_STATE_DIR))
+    Ok(ziggurat_path.join(STATEFUL_NODES_DIR).join("1"))
```

Comment:
`1` for the validator index? perhaps it should be `0` to match its token index?

Reply:
The testnet creates node directories like:
```
./ziggurat/ripple/testnet/1
./zigguratripple/testnet/2
./zigguratripple/testnet/3
```

Then these are copied to `./ziggurat/ripple/stateful/`
So we are referring here to the first copied stateful node from that folder: `./ziggurat/ripple/stateful/1/`